src/global.css

```diff
@@ -11,6 +11,7 @@ html {
 body {
   font-family: "Press Start 2P", cursive;
   margin: 0;
+  background-color: #545d66;
```

Comment:
Is there a reason why you put explicitly this background-color into the global CSS?